x/oracle/types/claim.go

```diff
@@ -0,0 +1,18 @@
+package types
+
+import sdk "github.com/cosmos/cosmos-sdk/types"
+
+type Claim struct {
+	ID               string         `json:"id"`
+	ValidatorAddress sdk.ValAddress `json:"validator_address"`
+	Content          string         `json:"content"`

```

Comment:
I think both ID and Content should be `[]byte`. It gives developers more freedom to customize their ID or Content and then just marhsal/unmarshal it when interacting with the oracle. It also creates easier validation since we can just say
```go
if len(ID) == 0 || len(Content) == 0 {
    return err
}
```
Otherwise is `"   "` considered valid? What do others think?

Reply:
well, a `string` is technically a `byte` array so I don't see the problem in that case

Reply:
I think it makes more sense to use `[]byte` since it is easier to manipulate byte arrays rather than strings

Reply:
The claim content is used for the ClaimValidators mapping below for constant time lookups. Golang doesn't suppert maps with []byte as the key, which is why string was used.